src/solver/conv_hip_implicit_gemm_fwd_v4r4_xdlops.cpp

```diff
@@ -142,15 +160,16 @@ void PerformanceImplicitGemmForwardV4R4Xdlops::EuristicInit(const ConvolutionCon
         }
         else if(ctx.IsFp16())
         {
-            tmp = {256, 256, 8, 128, 128, 8, false, true};
-
+            tmp              = {256, 256, 8, 128, 128, 8, false, true, 1};
```

Comment:
This heuristic would likely make the tuning parameter worse than before, because ```GemmBThreadDataPerRead_GemmN``` is always set to 1.

After discussion with @daniellowell, I think this would be an issue. Please call ```CalculateGemmBBlockCopyPerformancePara``` between line 230 and line 231 to determine what  ```GemmBThreadDataPerRead_GemmN``` should be.

Reply:
@daniellowell 

@jane-zxy corrected me, that if when ```MIOPEN_DEBUG_CONV_IMPLICIT_GEMM_FWD_V4R4_XDLOPS_ADD_VECTOR_LOAD_GEMMN_TUNE_PARAM=0```, which is true by default, then the heuristic tuning parameter will be the same as before. So this PR is fine